src/sentry/conf/server.py

```diff
@@ -941,6 +942,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     Queue("nudge.invite_missing_org_members", routing_key="invite_missing_org_members"),
     Queue("auto_resolve_issues", routing_key="auto_resolve_issues"),
     Queue("on_demand_metrics", routing_key="on_demand_metrics"),
+    Queue("check_new_issue_threshold_met", "check_new_issue_threshold_met"),
```

Comment:
Nit: Not sure if `routing_key="check_new_issue_threshold_met"` is required here but may as well keep things consistent.

Reply:
Also, if you're adding a new queue you'll need to talk to ops to make sure workers are defined (I always forget if new queues automatically just get bucketed with the default queue or not)

Reply:
Checked with ops that this will use workers from the default pool as expected.